tests/test_jointcal_hsc.py

```diff
@@ -57,12 +61,20 @@ def test_jointcalTask_2_visits(self):
         self._testJointCalTask(2, relative_error, absolute_error)
 
     @unittest.skipIf(data_dir is None, "validation_data_hsc not setup")
-    def test_jointcalTask_6_visits(self):
+    def test_jointcalTask_4_visits(self):
         # NOTE: The relative RMS limit was empirically determined from the
         # first run of jointcal on this data. We should always do better than
         # this in the future!
-        relative_error = 10e-3*u.arcsecond
-        self._testJointCalTask(6, relative_error, absolute_error)
+        relative_error = 17e-3*u.arcsecond
+        self._testJointCalTask(4, relative_error, absolute_error)
+
+    # @unittest.skipIf(data_dir is None, "validation_data_hsc not setup")
+    # def test_jointcalTask_6_visits(self):
+    #     # NOTE: The relative RMS limit was empirically determined from the
+    #     # first run of jointcal on this data. We should always do better than
+    #     # this in the future!
+    #     relative_error = 10e-3*u.arcsecond
+    #     self._testJointCalTask(6, relative_error, absolute_error)
```

Comment:
This is in flux right now because the hsc data set changed recently. I'm going to leave it for when I sort out how to run on the new hsc data (I don't remember which set of visits goes with which).